api/tests/integration/infrastructure/repositories/student-repository_test.js

```diff
@@ -0,0 +1,37 @@
+const { expect, databaseBuilder } = require('../../../test-helper');
+const studentRepository = require('../../../../lib/infrastructure/repositories/student-repository');
+
+describe('Integration | Infrastructure | Repository | student-repository', () => {
+
+  describe('#findReconciledStudentsByNationalStudentId', () => {
+
+    it('should return instances of Student', async () => {
+      // given
+      const firstNationalStudentId = '1234';
+      const firstUser = databaseBuilder.factory.buildUser();
```

Comment:
Je propose ça, comme ça le fait qu'un INE matche 2 comptes est mis en valeur (= c'est la même personne, quand on voit 2 users on peut penser que ce sont 2 personnes différentes)
```suggestion
      const firstAccount = databaseBuilder.factory.buildUser();
```